src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -3,10 +3,13 @@
 Azure.ApplicationModel.Configuration is a component of the .NET Azure SDK. 
 It provides APIs for Microsoft [Azure's App Configuration Service](https://docs.microsoft.com/en-us/azure/azure-app-configuration/).
 
-Developers can use this SDK to interact with their [Configuration Stores](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) in the following ways:
-- Perform basic reads, writes, updates, and deletes
-- Get key-value history
-- Watch for changes in a specific key-value
+Developers can use this SDK to interact with the [Configuration Store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-dotnet-core-app#create-an-app-configuration-store) where the configuration settings or key-value are stored.
```

Comment:
This sounds like either settings or key-values can be stored. Do we need to keep the "key-value" part? Why not say that "configuration settings can be stored".

Reply:
I guess we could. it will be explicit to our scenario